rear_camera_py/Makefile

```
@@ -60,6 +65,13 @@ kill:
 rm-img:
 	sudo rm -rf image_data/
 
+## ブロックエリア攻略関連 ##
+hunt-l:
+	cd src && python3 block_de_treasure_hunter.py left
+
+hunt-r:
+	cd src && python3 block_de_treasure_hunter.py right
```

```suggestion
hunt-l:
	python3 src/block_de_treasure_hunter.py left

hunt-r:
	python3 src/block_de_treasure_hunter.py right
```
わざわざ1個のコマンドで済むのに、2個のコマンドを合わせるのは無駄な気がするのでこっち派。
コマンドは小文字でないといけないとかある？LRの方がわかりやすいと思う